sdk/core/core/src/az_span.c

```diff
@@ -414,19 +407,20 @@ AZ_NODISCARD az_result _az_span_replace(az_span* self, int32_t start, int32_t en
 AZ_NODISCARD az_result az_span_append_dtoa(az_span destination, double source, az_span* out_span)
 {
   AZ_PRECONDITION_NOT_NULL(out_span);
+  AZ_RETURN_IF_SPAN_CAPACITY_TOO_SMALL(destination, 1);
```

Comment:
is `1` ok here?
for negative numbers, we need one byte for the `-` and then we need one extra byte for each digit right?

Reply:
I see,,  we add this again below once we know the digit count.
We might want to have a comment here about it